src/grpc/lightning.ts

```diff
@@ -599,7 +587,7 @@ export function signBuffer(msg: Buffer, ownerPubkey?: string): Promise<string> {
         const recidBytes = sigBuf.subarray(66, 67)
         // 31 is the magic EC recid (27+4) for compressed pubkeys
         const ecRecid = Buffer.from(recidBytes).readUIntBE(0, 1) + 31
-        const finalRecid = Buffer.from('00', 'hex')
+        const finalRecid = Buffer.allocUnsafe(1)
```

Comment:
not sure why we're changing this here does eslint not like the way we did it previously?

when I do this script its giving me false
```
var buf1 = Buffer.from('00', 'hex')
var buf = Buffer.allocUnsafe(1);
console.log(buf1 == buf);
console.log(buf1)
console.log(buf)
```

Reply:
not sure if I fully understand how Buffer works will need to look into it a bit

Reply:
its a minor change that runs faster looks cleaner.

your code snippet returns false because of javascript references (the buffers point to different places in memory despite being the same)

differences between buffer initialization methods:

`Buffer.from(string, 'hex')`: buffer with `string.length/2` length and contents of the string
`Buffer.alloc(lenght)`: buffer with `length` length filled with zeros (like c calloc)
`Buffer.allocUnsafe(length)`: buffer with `length` length, just allocated, not touched so can contain garbage (like c malloc)

the first one is the slowest, the last one the fastest.
the buffer is rewritten later in the code so allocUnsafe suits this case best

Reply:
gotcha that makes sense and yea we rewrite over soon after so this looks good 🚀